### ISSUE TITLE
Meeting notification uses username, not full name

### ISSUE THREAD

**Issue by andyshilton** [4minitz/4minitz#452]
When the email is sent out to participants to let them know of a new meeting, the "invited" section uses your full name. However, the top of the email seems to use the user's username. Could this be swapped to use full name instead so it's a bit more friendly?

The punctuation doesn't quite read right. While there's a comma on the end of the "Hello <<username>>", could this be removed and the next line start with a capital?

"We've got a brand new meeting agenda for you:"

![image](https://user-images.githubusercontent.com/16556349/41903846-d9f163ee-792e-11e8-8cc1-fae4fb135891.png)

**Comment by derwok** [comment]
Hi Andy.
Thanks for reporting. Yes, we can try to fix this.
We must see, if the user long name is easyly available in the eMail template.

The "Comma" and "Capital" topic comes from our German speaking roots.
In Germany it is best practice to place a comma behind the greeting line and then continue with a small letter. But you're right - English is different here.

**Comment by derwok** [comment]
Hi Andy,
just a quick note on this fix.
While we could fix the "Comma" and "Capital" topic - your main issue is more complex.

Let me explain: 4Minitz supports two modes in settings.json: trustedIntranetInstallation = true or false.
1.) trustedIntranetInstallation=false
When someone runs a "public" server, where people can self-register, we considered it a possible security leak, if some (evil) one can see the email addresses or long names of other users. Thats why when a meeting agenda or meeting minutes are sent out via mail - each user gets HIS/HER OWN separate email. Only in this case we can say "Hello John Doe". For simplicity we use the username here... And we won't change this, as the untrusted mode is not the MAIN use case of 4Minitz.

2.) trustedIntranetInstallation=true
This should be the standard run mode of 4Minitz. Inside a company intranet, users come from LDAP and usually no self-registering is possible. So in this mode when an agenda or meeting minutes email is sent, we send out ONE phyisical mail that is sent with all invited people. The main advantage here are: A) all users can see who also got the same mail and B) if one user has some feedback (e.g., additional agenda topic OR correction to the meeting minutes) HE/SHE can simply use the mail client via "Reply-To-All". For obvious reasons we would not allow this cool feature in an untrusted environment.
In this case (where all users get the phyisical same eMail to allow a reply-to-all), we can't put the users name to the "Hello"-greetings line since we must put ALL users name there - this looks ood.

**So, long story short**: because you see the "Hello andy.shilton" in your EMails this tells us that you probably set `trustedIntranetInstallation=false` which is probably not optimal. It prohibits your users to use the "Reply-To-All" feature. If you have either protected 4Minitz from the outside world via firewall OR if you have switched of self registering (`"forbidClientAccountCreation": true`) - the you should set `"trustedIntranetInstallation": true`. Then also the strange "Hello" greetings will be gone. You will only see "Hello" then (without a name).

Complicated. Sorry for that...
But I hope you get the point.

**Comment by andyshilton** [comment]
Hi,

Thank you for the explanation, and taking the time to be so detailed about it too. It does all make sense. It sounds like, in short, we need to try the trusted intranet installation :)

Andy.

**Comment by wasimdf** [comment]
hi...team can we enable email notifications for overdue action items?
